gradlew

```diff
@@ -1,25 +1,43 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
Was this update to gradlew on purpose?

Reply:
These were all automatic gradle updates that AS handled.